lib/lua/addDelayedJob.lua

```diff
@@ -1,5 +1,5 @@
 --[[
-key 1 -> bq:name:id (job ID counter)
+key 1 -> bq:name:id (job counter)
```

Comment:
What is stored in `bq:name:id`? My understanding from this PR is that it's the number of jobs that don't have a custom id? What's that used for?

Reply:
I believe that is correct, it's poorly named, and appears to come from the fact that auto-generated IDs are currently sequential. So job ID 1000 would be the 1000th job. This PR of course changes that but does not rename to try to maximize compatibility, replacing the stat with the actual count instead of the actual last job ID from which you would infer the count.

Reply:
Where is that information ("the number of jobs that don't have a custom id") used?